src/extension.ts

```diff
@@ -62,4 +116,21 @@ export function activate(context: vscode.ExtensionContext) {
   const treeView = vscode.window.createTreeView('vscodeMarkdownNotesBacklinks', {
     treeDataProvider: backlinksTreeDataProvider,
   });
+
+  // See: https://code.visualstudio.com/api/extension-guides/markdown-extension
+  // For more information on how this works.
+  return {
+    extendMarkdownIt(md: any) {
```

Comment:
Please move a wrapper around this into the new function that you create.

Reply:
:+1: